### DISCUSSION TITLE
Newbie in StackStorm

### DISCUSSION THREAD

**Discussion by idham1311** [StackStorm/st2#5771] [General]
I'm still new in StackStorm. Do you hv any suggestions from where I need to start?

**Comment by arm4b** [comment]
You can join the https://stackstorm.com/#community in Slack, Install StackStorm https://docs.stackstorm.com/install/index.html and start with some basics https://docs.stackstorm.com/start.html.

Try to solve some simple automation problem with StackStorm in your environment.
Feel free to ask any questions in Slack, - the community will give some direction and pointers if you're stuck.

**Reply by idham1311** [reply]
Thank you :)
